types/next-server/index.d.ts

```diff
@@ -0,0 +1,121 @@
+// Type definitions for next-server 7.0
+// Project: https://github.com/zeit/next.js/packages/next-server
+// Definitions by: AJ Livingston <https://github.com/ajliv>
```

Comment:
Please don't forget to add the other maintainers from `next` here. And add your name there as well.